### ISSUE TITLE
Cmd + dot is interpreted as keyboard Esc key; can it be Esc + dot instead?

### ISSUE THREAD

**Issue by gcv** [blinksh/blink#1473]
Cmd + dot on a hardware keyboard seems to send the same thing to Blink as the physical Esc key. This breaks using something like `M-.` in Emacs. I suspect this happens somewhere deep in the OS, but maybe a workaround is possible?

**Comment by LoganDark** [comment]
In my experience, Meta typically means Alt, not Command. Does Blink translate Command to Meta or?

**Comment by yury** [comment]
@gcv sorry for late response. Blink sends `^[.` physical `esc` produce just `^[`. You can test it in showkey command.

**Comment by gcv** [comment]
I looked into this a little deeper, and I have a bit more information about the problem.

Adding a Custom Press for Combination Esc to Action Press <kbd>`</kbd> causes the Blink shell to interpret Cmd-dot as a backtick character and print a backtick to the terminal. This happens without logging into anything, just at the level of the Blink command prompt. Hence the confusing ticket title: with that custom press setting, it looks like Blink interprets Cmd-dot as the Esc key, although that’s probably not what actually happens.

I also have the Command key configured in Blink as follows:
- same for both sides: false
- left press send: none
- left as modifier: Esc
- right press send: none
- right as modifier: default

I observe the same behavior with both left and right Cmd keys, i.e. they both print backticks when I press Cmd-<kbd>.</kbd>.

Background information: I have a hardware keyboard which does not have a dedicated key for backtick and tilde characters; this physical key is instead shared with Esc and backtick/tilde require a Fn1 modifier ([Keychron K7](https://www.keychron.com/pages/keychron-k7-wireless-mechanical-keyboard)). This causes a nuisance because I (personally) need backtick and tilde more frequently than I need Esc. So I made a mapping in Blink so that pressing Esc is a backtick, and Shift-Esc is a tilde. This works great, except it has an unexpected side effect with Cmd.
